src/helpers/mathjax.ts

```diff
@@ -95,9 +95,24 @@ function typesetDocument(root: Element, windowImpl: Window) {
   );
 }
 
+const resolveOrWait = (root: Element, resolve: () => void, remainingTries = 5) => {
+  if (
+    remainingTries
```

Comment:
i would make this 
```suggestion
    remainingTries > 0
```
for reduced possibility of infinite recursion (negative numbers are truthy)